tests/aws/services/cloudformation/resources/test_apigateway.py

```diff
@@ -212,7 +212,13 @@ def test_cfn_with_apigateway_resources(deploy_cfn_template, aws_client, snapshot
 
 @markers.aws.validated
 @markers.snapshot.skip_snapshot_verify(
-    paths=["$.get-resources.items..resourceMethods.ANY"]  # TODO: empty in AWS
+    paths=[
+        "$.get-resources.items..resourceMethods.ANY",  # TODO: empty in AWS
+        "$..requestParameters",  # FIXME: it seems AWS does not return empty dicts anymore, will need to fix
+        "$..responseTemplates",  # FIXME: it seems AWS does not return empty dicts anymore, will need to fix
```

Comment:
:face_exhaling:  of course. guess that will affect a lot of apigw snapshots

Reply:
Yes, I'll take some time next week to try to regenerate most of them and adapt the code, mostly moto patches, oh joy 😄